references/classification/train.py

```diff
@@ -80,20 +80,21 @@ def _get_cache_path(filepath):
     cache_path = os.path.expanduser(cache_path)
     return cache_path
 
-
 def main(args):
+    if args.output_dir:
+        utils.mkdir(args.output_dir)
```

Comment:
Why was this code block moved?

Reply:
I wanted to include `torch.cuda.set_device(args.gpu)` into apex initialization. This needs to be done after `utils.init_distributed_mode(args)` as args don't have `args.gpu` initially. But since we don't need this anymore, apex initialization code can be moved back to the beginning of main()